site/en/r2/tutorials/load_data/text.ipynb

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
<p>Remove the first sentence (This model running on this data produces decent results (about 83%) after just two epochs and levels off quickly. A third epoch lowers accuracy by a tiny amount.)</p><p><br></p><p>Put this sentence in Next Steps (If you edit the model, try running several epochs to see when improvement levels off and overfitting sets in.)</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>